### PR TITLE
Test Agent/Client CAN modification

### PR DESCRIPTION
**Pull request by Acuadros95**

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/308/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/328/)

**Review by pablogs9 — DISMISSED** [review]

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/310/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/330/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/311/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/331/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/312/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/332/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/313/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/333/)

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/319/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/339/)

File: CMakeLists.txt [hunk 1]
```diff
@@ -29,9 +29,9 @@ endif()
 
 project(microxrcedds VERSION "2.1.0")
 
-set(_client_tag v2.1.0)
+set(_client_tag fix/can_len)
```
Comment (Acuadros95) [inline CMakeLists.txt:32]:
```suggestion
set(_client_tag develop)
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -29,9 +29,9 @@ endif()
...
 set(_client_version 2.1.0)
-set(_agent_tag v2.1.0)
+set(_agent_tag fix/can_len)
```
Comment (Acuadros95) [inline CMakeLists.txt:34]:
```suggestion
set(_agent_tag develop)
```

**Comment by richiprosima** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Linux/320/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS%20Manual%20Windows/340/)

**Review by pablogs9 — APPROVED** [review]

**Pull request merged**